### PR TITLE
Update mnemonics names vehicles and aftermarkets.

### PR DESCRIPTION
**Pull request by KevinJoiner**
Both mnemonic names are now bip39 compliant.
And encode and obfuscate tokenID.

### CODE REVIEW

**Comment by elffjs** [comment]
We're going to move this library to `DIMO-Network`, I think.

**Comment by elffjs** [comment]
Where are we picking the prime? Is that fixed in the library?

**Comment by KevinJoiner** [comment]
> Where are we picking the prime? Is that fixed in the library?

@elffjs  It is a constant in the mnemonic lib https://github.com/DIMO-Network/mnemonic/blob/main/mnemonic.go#L20

The hider is exposed so the prime used could be changed if someone so desired.
https://github.com/DIMO-Network/mnemonic/blob/main/mnemonic.go#L29

File: internal/repositories/aftermarket/aftermarket_devices.go [hunk 1]
```diff
@@ -180,7 +181,8 @@ func AftermarketDeviceToAPI(d *models.AftermarketDevice, imageUrl string) *gmode
 
 	_ = e.Encode(aftermarketDevicePrimaryKey{TokenID: d.ID})
 
-	name, _ := helpers.CreateMnemonic(d.Address)
+	nameList := mnemonic.FromInt64WithObfuscation(int64(d.ID))
```
[the remark this answers is not in the document]
Comment (elffjs) [reply internal/repositories/aftermarket/aftermarket_devices.go:184]: Okay it is 6, I'm not crazy. 

https://go.dev/play/p/ryDFXz_QgQW

The ids are integers in Postgres, so I think they're really 32-bit signed integers.

[the remark this answers is not in the document]
Comment (KevinJoiner) [reply internal/repositories/aftermarket/aftermarket_devices.go:184]: That works int32 will fit into 3 words.

**Review by elffjs — APPROVED** [review]

**Comment by elffjs** [comment]
@Allyson-English Can you give this another look?

**Comment by elffjs** [comment]
Ah crap we got a little merge conflict.

**Comment by KevinJoiner** [comment]
Did some rebasing to fix the merge conflict.

**Comment by elffjs** [comment]
@Allyson-English Giving this a nudge.

**Pull request merged**